SimPEG/Maps.py

```diff
@@ -1852,7 +2017,7 @@ def __init__(self, mesh, pts, ptsv=None, order=3, logSigma=True,
 
 class BaseParametric(IdentityMap):
 
-    slopeFact = 1e2  # will be scaled by the mesh.
+    slopeFact = 1  # will be scaled by the mesh.
```

Comment:
this changes a default - here, the slope factor should not be as severe ... makes the parametric maps converge more slowly